README.md

```diff
@@ -5,7 +5,7 @@ easily.
 
 RQ requires Redis >= 3.0.0.
 
-[![Build status](https://travis-ci.org/rq/rq.svg?branch=master)](https://secure.travis-ci.org/rq/rq)
+![Test rq](https://github.com/aparcar/rq/workflows/Test%20rq/badge.svg)
```

Comment:
Link to the badge should be set to RQ's repo. Can we also change this to `Build status`?

Reply:
Oops, no self promotion intended here.

Reply:
And it's also renamed to `Build status` now.